added Profile section to main readme file (#394)
mongodb/mongodbatlas-cloudformation-resources: README.md

Support for the MongoDB Atlas Resource Provider for CloudFormation is provided under MongoDB Atlas support plans, starting with Developer. Please submit support questions within the Atlas UI. In addition, support questions submitted under the Issues section of this repo are also being monitored. Bugs should be filed under the Issues section of this repo.

> # Atlas API Keys Credential Management

Reviewer: ```suggestion
# MongoDB Atlas API Keys Credential Management
```

> Atlas API keys Configuration re required both cloud formation and CDK resources, and this key pair are provided as input by the use of a Profile

Reviewer: ```suggestion
Atlas API keys Configuration required both cloud formation and CDK resources, and this key pair are provided as input by the use of a Profile
```

> Atlas API keys Configuration re required both cloud formation and CDK resources, and this key pair are provided as input by the use of a Profile

Reviewer: ```suggestion
Atlas API keys Configuration re required both cloud formation and CDK resources, and this key pair are provided as input by the use of a Profile to configure the API keys and a secret profile follow the next steps:
```

> to configure the API keys and a secret profile follow the next steps:

Reviewer: ```suggestion
```

> ## 1. Configure your ATLAS API KEYS

Reviewer: ```suggestion
## 1. Configure your MongoDB Atlas API Keys 
```

You'll need to generate an API key pair (public and private keys) for your Atlas organization and configure them to grant CloudFormation access to your Atlas project.
Refer to the [Atlas documentation](https://www.mongodb.com/docs/atlas/configure-api-access/#manage-programmatic-access-to-an-organization) for detailed instructions.

## 2. Configure your Profile

> To use Atlas CloudFormation resources, you must configure a "profile" with your API keys using AWS Secrets Manager.

Reviewer: ```suggestion
To use Atlas CloudFormation resources, you must configure a "profile" with your API keys using [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).
```

The secret should follow this format:
```
SecretName: cfn/atlas/profile/{ProfileName}
SecretValue: {PublicKey: {YourPublicKey}, PrivateKey: {YourPrivateKey}}
```

To create a new secret for a default profile, use the [PROFILE SECRET TEMPLATE](/examples/profile-secret.yaml) file provided in this repository.

Here are some examples of how to use this template:

### example 1:
```
  ProfileName: default
  SecretName: cfn/atlas/profile/default
  SecretValue = {PublicKey: xxxxxxx , PrivateKey: yyyyyyyy}
```
### example 2:
```
  ProfileName: tetProfile1
  SecretName: cfn/atlas/profile/tetProfile1
  SecretValue = {PublicKey: zzzzzzzzzz , PrivateKey:jjjjjjjjj}
```

> ## 3 Provide the profile to your Cloud Formation template

Reviewer: ```suggestion
## 3. Provide the profile to your Cloud Formation template
```

All Atlas CloudFormation resources include a "Profile" property that specifies which profile to use. You'll need to provide the profile you created in the previous step to the CloudFormation template.

Note that if you don't provide a profile, the resource will use a default profile (will try to get a secret named cfn/atlas/profile/default). We recommend always specifying the profile to avoid any unexpected behavior.

Once you've provided the profile, you can deploy the CloudFormation stack using the AWS Console or the AWS CLI. Refer to the AWS documentation for instructions on how to deploy CloudFormation stacks.

> IMPORTANT: when specifying the profile in your cloud formation template, you must specify the Profile Name, NOT the Secret Name

Reviewer: ```suggestion
IMPORTANT: when specifying the profile in your CloudFormation template, you must specify the Profile Name, NOT the Secret Name
```
